commit 3b1047181fbbbd2067b6b7476c428199474fdd19
Author: jochenberger <fooberger@gmail.com>
Date:   Thu Apr 7 21:38:46 2016 +0200

    Enable hid-betopff module
    
    Add force feedback support for Betop based devices
    https://github.com/raspberrypi/linux/blob/rpi-4.1.y/drivers/hid/hid-betopff.c

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 7bfc073..31e3ac5 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -867,6 +867,7 @@ CONFIG_HID_A4TECH=m
 CONFIG_HID_ACRUX=m
 CONFIG_HID_APPLE=m
 CONFIG_HID_BELKIN=m
+CONFIG_HID_BETOP_FF=m
 CONFIG_HID_CHERRY=m
 CONFIG_HID_CHICONY=m
 CONFIG_HID_CYPRESS=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 59026b3..ac9287f 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -859,6 +859,7 @@ CONFIG_HID_A4TECH=m
 CONFIG_HID_ACRUX=m
 CONFIG_HID_APPLE=m
 CONFIG_HID_BELKIN=m
+CONFIG_HID_BETOP_FF=m
 CONFIG_HID_CHERRY=m
 CONFIG_HID_CHICONY=m
 CONFIG_HID_CYPRESS=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1237_3b1047181fbbbd2067b6b7476c428199474fdd19.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1237_3b1047181fbbbd2067b6b7476c428199474fdd19.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
